components/LiveAnalytics.tsx

```diff
@@ -1,12 +1,14 @@
 import Script from "https://deno.land/x/partytown@0.1.0/Script.tsx";
-import { Page, Site } from "$live/types.ts";
+import { Flags, Page, Site } from "$live/types.ts";
 
-const innerHtml = ({ id, path }: Page, { id: siteId }: Site) => `
+const innerHtml = ({ id, path }: Page, { id: siteId }: Site, flags: Flags) => `
 import { onCLS, onFID, onLCP } from "https://esm.sh/web-vitals@3.1.0";
 
 function onWebVitalsReport(event) {
   if (typeof window.jitsu === "function") {
-    window.jitsu('track', 'web-vitals', { ...event, page_id: "${id}", page_path: "${path}", site_id: "${siteId}" });
+    window.jitsu('track', 'web-vitals', { ...event, page_id: "${id}", page_path: "${path}", site_id: "${siteId}", ${
+  Object.keys(flags).map((key) => `flag_${key}: true`).join(",")
+} });
   }
```

Comment:
There should be another way of injecting this info. I think flags are more "global" than web vitals and should be registered to the section. Something like [.set](https://jitsu.com/docs/sending-data/js-sdk/reference/methods-reference#set)

Reply:
Ohh, my bad, I didn't notice this was a `web-vitals` specific event, d'oh. So much for programming late at night. I just stuffed it anywhere in sight 😂